build/rakelib/cluster.rake

```diff
@@ -119,6 +119,42 @@ namespace :cluster do
     end
   end
 
+  desc "Deploy ebrelayer"
+  namespace :ebrelayer do
+    task :deploy, [:chainnet, :provider, :namespace, :image, :image_tag, :ethereum_websocket_address, :ethereum_contract_address, :ethereum_private_key] do |t, args|
+      check_args(args)
+
+      cmd = %Q{helm upgrade sifnode ../build/helm/sifnode \
+        --set sifnode.env.chainnet=#{args[:chainnet]} \
+        --install -n #{ns(args)} \
+        --set image.tag=#{image_tag(args)} \
+        --set image.repository=#{image_repository(args)} \
+        --set ebrelayer.enabled=true \
+        --set ebrelayer.image.tag=#{image_tag(args)} \
+        --set ebrelayer.env.ethereumWebsocketAddress=#{args[:ethereum_websocket_address]} \
+        --set ebrelayer.env.ethereumContractAddress=#{args[:ethereum_contract_address]} \
+        --set ebrelayer.env.moniker=#{args[:chainnet]} \
+        --set ebrelayer.env.ethPrivateKey=#{args[:ethereum_private_key]}
+      }
+
+      system({"KUBECONFIG" => kubeconfig(args) }, cmd)
+    end
+
+    task :uninstall, [:chainnet, :provider, :namespace] do |t, args|
+      check_args(args)
+
+      cmd = %Q{helm upgrade sifnode ../build/helm/sifnode \
```

Comment:
this is an `upgrade` call to helm... However wont a uninstall remove the whole sifnode install??

Reply:
@utx0 In this step I am only uninstalling `ebrelayer` by setting the `enabled` flag with false and the deployment will be removed from the cluster, I am not making any changes in the `sifnode` deployment becuase we have separate uninstall task for destroying `sifnode`.

Reply:
ok thanks for that. I didnt notice the false flag at first.